src/main/java/address/browser/BrowserManager.java

```diff
@@ -101,7 +100,8 @@ public synchronized void loadProfilePage(ReadOnlyViewablePerson person) {
         try {
             Page page = hyperBrowser.get().loadUrls(person.profilePageUrl(), listOfFutureUrl);
             GithubProfilePage gPage = new GithubProfilePage(page);
-            gPage.setPageLoadFinishListener(b -> Platform.runLater(() -> gPage.activateAutomateClickingAndScrolling()));
+            gPage.executePageLoadedTasks();
```

Comment:
If the call below (`setPageLoadFinishListener`) is not enough, then sure. No more comments.